### ISSUE TITLE
PALI pretrained model

### ISSUE THREAD

**Issue by Debolena7** [kyegomez/PALI#5]
Which model size do we get from here?

import torch
from pali import Pali

**Comment by kyegomez** [comment]
@not-hermione This is not a pretrained model this is the model architecture! It's not a big model, you can change the size by increasing the dim and patch size, etc!
